packages/seo-integration/package.json

```diff
@@ -2,10 +2,34 @@
   "name": "@yoast/seo-integration",
   "version": "1.0.0",
   "jest": {
-    "testRegex": "src\\/.+\\/(?:.*[-.])?tests?\\.js$",
+    "collectCoverage": true,
+    "coverageThreshold": {
```

Comment:
For the reviewers:
The coverage thresholds here only serve as an initial number for now. These thresholds may be adjusted in the future once we add more unit tests inside the package.

Reply:
Do we want to collect coverage from all the source files, or only for those that we have tests for?

Right now it is configured to only collect coverage from the source files we have unit tests for.

This may give us a wrong idea of the total coverage of the package. Since entirely untested files do not show up in the coverage report.

Compare
```
-------------------|---------|----------|---------|---------|----------------------------------------
File               | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s                      
-------------------|---------|----------|---------|---------|----------------------------------------
All files          |   43.48 |      100 |    7.14 |   43.48 |                                        
 configurations.js |   43.48 |      100 |    7.14 |   43.48 | 15-16,21-22,35,46-47,52-53,59-60,65-66 
-------------------|---------|----------|---------|---------|----------------------------------------
```
with
```
--------------------------------|---------|----------|---------|---------|----------------------------------------
File                            | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s                      
--------------------------------|---------|----------|---------|---------|----------------------------------------
All files                       |    2.82 |     1.98 |    0.88 |    3.32 |                                        
 src                            |       0 |        0 |       0 |       0 |                                        
  analysis.js                   |       0 |        0 |       0 |       0 | 16-64                                  
  analyze.js                    |       0 |        0 |       0 |       0 | 14-166                                 
  google-preview-container.js   |       0 |        0 |       0 |       0 | 16-147                                 
  index.js                      |       0 |        0 |       0 |       0 | 46-68                                  
  seo-context.js                |       0 |      100 |       0 |       0 | 4-40                                   
 src/analysis-result-containers |       0 |        0 |       0 |       0 |                                        
  hooks.js                      |       0 |        0 |       0 |       0 | 10-21                                  
  index.js                      |       0 |        0 |       0 |       0 |                                        
  readability.js                |       0 |        0 |       0 |       0 | 16-46                                  
  seo.js                        |       0 |        0 |       0 |       0 | 18-52                                  
  transformers.js               |       0 |        0 |       0 |       0 | 36-97                                  
 src/containers                 |       0 |        0 |       0 |       0 |                                        
  facebook-preview-container.js |       0 |      100 |       0 |       0 | 4-6                                    
  google-preview-container.js   |       0 |        0 |       0 |       0 | 16-147                                 
 src/containers/analysis        |       0 |        0 |       0 |       0 |                                        
  hooks.js                      |       0 |        0 |       0 |       0 | 10-21                                  
  index.js                      |       0 |        0 |       0 |       0 |                                        
  readability.js                |       0 |        0 |       0 |       0 | 16-46                                  
  seo.js                        |       0 |        0 |       0 |       0 | 18-52                                  
  transformers.js               |       0 |        0 |       0 |       0 | 36-97                                  
 src/replacement-variables      |   27.78 |      100 |    5.26 |   28.57 |                                        
  configurations.js             |   43.48 |      100 |    7.14 |   43.48 | 15-16,21-22,35,46-47,52-53,59-60,65-66 
  index.js                      |       0 |      100 |       0 |       0 | 12-60                                  
--------------------------------|---------|----------|---------|---------|----------------------------------------
```
We can tell Jest to collect coverage from all files by adding this line to the Jest settings:
```json
"collectCoverageFrom": [
   "./src/**/*.js"
],
```

Reply:
Good point! yes, I think it makes sense to have the reports for all the files in the package. We do the same in `yoastseo` package. I'll implement this suggestion 😺